app/assets/src/components/AlignmentViz.jsx

```diff
@@ -28,13 +29,21 @@ class AlignmentViz extends React.Component {
       )
       .then(res => {
         this.setState({
-          alignmentData: res.data
+          alignmentData: res.data,
+          loading: false
         });
       });
   }
 
   render() {
-    return (
+    return this.state.loading ? (
+      <div>
+        {" "}
+        <h2>
+          Loading alignment data for {this.taxName} ({this.taxLevel}) ...{" "}
```

Comment:
For the future you can also write this as:

<h2>
  {` Loading alignment data for ${this.taxName} (${this.taxLevel}) ... `}
</h2>